apiserver/params/storage.go

```diff
@@ -33,3 +33,28 @@ type StorageShowResult struct {
 type StorageListResult struct {
 	Instances []StorageInstance
 }
+
+// PoolInstance holds data for a pool instance.
+type PoolInstance struct {
```

Comment:
s/PoolInstance/StoragePool/

"Pool" isn't very informative without the storage context, and "instance" isn't meaningful here. Pools are always instances.